tasks/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
-- name: Install pip
-  apt: pkg=python-pip={{ pip_version }} state=present
+- name: Download get-pip.py
+  get_url: url=https://bootstrap.pypa.io/get-pip.py
+           dest=/tmp/get-pip.py
+  notify: Install pip
```

Comment:
It was a suggestion from ansible-lint to maintain idempotency. It keeps us from having to add additional logic to decide if we should run the `install` task.